include/z64save.h

```diff
@@ -572,18 +572,38 @@ typedef enum LinkAge {
 #define EVENTCHKINF_PAID_BACK_BUNNY_HOOD 0x8F
 
 // EVENTCHKINF 0x90-0x93
-// carpenters freed from the gerudo
-#define EVENTCHKINF_INDEX_CARPENTERS_FREE 9
-#define EVENTCHKINF_CARPENTERS_FREE_SHIFT(n) (0 + (n))
-#define EVENTCHKINF_CARPENTERS_FREE_MASK(n) (1 << EVENTCHKINF_CARPENTERS_FREE_SHIFT(n))
-#define EVENTCHKINF_CARPENTERS_FREE(n) ((EVENTCHKINF_INDEX_CARPENTERS_FREE << 4) | EVENTCHKINF_CARPENTERS_FREE_SHIFT(n))
-#define EVENTCHKINF_CARPENTERS_FREE_MASK_ALL (\
-      EVENTCHKINF_CARPENTERS_FREE_MASK(0)     \
-    | EVENTCHKINF_CARPENTERS_FREE_MASK(1)     \
-    | EVENTCHKINF_CARPENTERS_FREE_MASK(2)     \
-    | EVENTCHKINF_CARPENTERS_FREE_MASK(3)    )
-#define GET_EVENTCHKINF_CARPENTERS_FREE_ALL() \
-    CHECK_FLAG_ALL(gSaveContext.save.info.eventChkInf[EVENTCHKINF_INDEX_CARPENTERS_FREE], EVENTCHKINF_CARPENTERS_FREE_MASK_ALL)
+// Carpenters rescued from Gerudo Fortress
+#define EVENTCHKINF_INDEX_CARPENTERS_RESCUED EVENTCHKINF_INDEX(EVENTCHKINF_CARPENTER_0_RESCUED)
```

Comment:
I understand why you did it this way, but it feels weird that only `EVENTCHKINF_CARPENTER_0_RESCUED` is used here and not the other carpenter flag IDs. Maybe it would be better to hardcode 0x9 here?

Reply:
Personally I prefer hardcoding all of the index values

Reply:
oh, why did you decide not to here?

Reply:
because I've been indecisive on which way to go. From now on it'll be consistent.

Reply:
It'll be consistent with not hardcoding the values?